v2/types.go

```diff
@@ -370,3 +370,20 @@ type UnbindRequest struct {
 type UnbindResponse struct {
 	// Currently, unbind responses have no fields.
 }
+
+// GetBindingRequest represents a request to do a GET on a particular binding.
```

Comment:
The service needs to have a field added to it that says whether you can get the binding for a service.

Reply:
Oh yeah, thanks. Added.